ush/wave_outp_spec.sh

```diff
@@ -175,7 +174,7 @@ EOF
      fi
    fi
   else
-    echo "ERROR: OUTPUT DATA FILE FOR BUOY ${buoy} NOT FOUND"
+    echo "FATAL ERROR: OUTPUT DATA FILE FOR BUOY ${buoy} NOT FOUND"
     exit 5
```

Comment:
`err_exit`?

Reply:
We're adopting the standard of calling `err_exit` at the ex-script and J-Job level.  There are a few functions defined in `ush` that are directly called by the ex-scripts.  These should contain `err_exit` for fatal errors, but all other ush scripts should exit with non-zero status and a descriptive `FATAL ERROR` message.

Reply:
Gotcha, thanks!